sdk/cmake/CMakeMacros.cmake

```diff
@@ -577,8 +577,11 @@ function(set_module_type MODULE TYPE)
         add_target_link_flags(${MODULE} "-Wl,--major-os-version,5 -Wl,--minor-os-version,01")
     endif()
 
-    # Set unicode definitions
+    # Set unicode property
     if(__module_UNICODE)
+        set_target_properties(${MODULE} PROPERTIES UNICODE_MODULE TRUE)
+        # This breaks IDL macros
```

Comment:
What kind of error(s) does it give?

Reply:
Something like "generator expressions can't be used for custom command"